src/components/nav-bar/NavBar.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { useHistory, useLocation } from 'react-router-dom';
+import { PageHeader, Typography, Button, Avatar, Menu, Dropdown } from 'antd';
+import { UserOutlined } from '@ant-design/icons';
+import './nav-bar.less';
+const { Paragraph } = Typography;
+
+const NavBar: React.FC = () => {
+  const history = useHistory();
+  const location = useLocation();
+  const path = location.pathname;
+
+  const isLoggedIn = () => {
+    return !(path === '/' || path === '/login' || path === 'signup');
+  };
```

Comment:
This is a bit hacky, let's leave a comment to update it to use a redux selector

Reply:
Actually, you could write such a selector. Ping me if you have questions on how that works, I'm writing docs now.